client/src/app/+admin/config/edit-custom-config/edit-custom-config.component.html

```diff
@@ -221,6 +221,42 @@
             </div>
           </div>
 
+          <div class="form-group">
```

Comment:
The transcoding concurrency included in the client is a bad idea cause it require a restart of the server to work :
https://github.com/Chocobozzz/PeerTube/blob/35d50b7dd26b3cf646b8845784927bb1ef18dfb3/server/lib/job-queue/job-queue.ts#L78 (as you can see transcoding concurrency is set at the init of the server).
Maybe we should keep it in the server but not in client :wink:

Reply:
Oh right nice catch @Jorropo (sorry @McFlat I forgot this)

Reply:
No worries, I'll try to get all these things done today.